src/Fable.Core/Fable.Core.Types.fs

```diff
@@ -2,6 +2,58 @@ namespace Fable.Core
 
 open System
 
+/// <summary>
+/// Parameter used in <c>StringEnum</c> and methods/other attributes that
+/// generate strings from unions to change the casing in conversion.
+/// </summary>
+/// <remarks>
+/// Conversion of <c>MouseOver</c> with different enums (the qualified access is
+/// ignored for brevity):
+/// <list type="table">
+/// <item>
+/// <term><c>None</c></term>
+/// <description><c>"MouseOver"</c></description>
+/// </item>
+/// <item>
+/// <term><c>LowerFirst</c></term>
+/// <description><c>"mouseOver"</c></description>
+/// </item>
+/// <item>
+/// <term><c>SnakeCase</c></term>
+/// <description><c>"mouse_over"</c></description>
+/// </item>
+/// <item>
+/// <term><c>SnakeCaseAllCaps</c></term>
+/// <description><c>"MOUSE_OVER"</c></description>
+/// </item>
+/// <item>
+/// <term><c>KebabCase</c></term>
+/// <description><c>"mouse-over"</c></description>
+/// </item>
+/// <item>
+/// <term><c>LowerAll</c></term>
+/// <description><c>"mouseover"</c></description>
+/// </item>
+/// </list>
```

Comment:
I need to check why but the rendering of this element is broken on Ionide.

I think you wanted to render a table but that's not the case for Ionide